tests/lib/rules/no-multiple-empty-lines.js

```diff
@@ -309,6 +309,11 @@ ruleTester.run("no-multiple-empty-lines", rule, {
             errors: [getExpectedError(1)],
             options: [{ max: 1 }],
             parserOptions: { ecmaVersion: 6 }
+        },
+        {
```

Comment:
Perhaps we could link to the related issue here, to make sure we don't forget about it?